File: PhotoFox.Storage/Models/UploadMessage.cs
using System;

namespace PhotoFox.Storage.Models
{
    public class UploadMessage
    {
        public UploadMessage()
        {
            this.Type = "UNKNOWN";
            this.EntityId = Guid.Empty.ToString();
            this.Album = string.Empty;
            this.Title = string.Empty;
            this.FileExt= string.Empty;
        }

        public string Type { get; set; }
        public string EntityId { get; set; }
        public string Album { get; set; }
        public string Title { get; set; }
        public DateTime DateTaken { get; set; }
        public string FileExt { get; set; }
    }
}

File: Photofox.Android/app/src/main/java/com/fionasapphire/photofox/storage/entity/UploadMessage.kt
package com.fionasapphire.photofox.storage.entity

import java.util.*

data class UploadMessage (
    val EntityId: String,
    val Type: String,
    val Album: String,
    val Title: String,
    val DateTaken: String,
    val FileExt: String
)
